tests/stress/stress.c

```diff
@@ -584,11 +583,19 @@ static int iterate_callback(struct eblob_disk_control *dc,
 						item->item->size, dc->data_size);
 				}
 				assert(item->item->size > 0);
+				void *data = malloc(item->item->size);
+				assert(data);
+				error = pread(fd, data, item->item->size, data_offset);
+				if (error == -1) {
+					errx(EX_SOFTWARE, "pread has been failed for: %s (%s), flags: %s, error: %d",
+					     item->item->key, eblob_dump_id(item->item->ekey.id), item->item->hflags, errno);
+				}
 				error = memcmp(data, item->item->value, item->item->size);
 				if (error != 0) {
 					errx(EX_SOFTWARE, "data verification has been failed for: %s (%s), flags: %s",
 					    item->item->key, eblob_dump_id(item->item->ekey.id), item->item->hflags);
 				}
+				free(data);
```

Comment:
Did you run eblob_stress with this whole new code?
Were there any errors? What were the options?